aol-htb-validator.js

```diff
@@ -69,11 +69,34 @@ var partnerValidator = function (configs) {
         }
     }, configs);
 
-    if (!result.valid) {
-        return result.format();
+    var aolMobileConfig = Inspector.validate({
```

Comment:
Can you please combine the aolDisplayConfig and aolMobileConfig validator together into one? 

You can use the exec function (https://github.com/Atinux/schema-inspector#v_exec) to make sure the xSlots have either `dcn + pos` or `placementId`

Reply:
I don't thins it is possible in this case because  "exec" function takes schema and value as params. But there is no access to validated object itself. We can't be sure that it has *dcn + pos*.